src/open_inwoner/accounts/views/cases.py

```diff
@@ -306,25 +306,6 @@ def get_context_data(self, **kwargs):
                 status_type = status_types_mapping[status.statustype]
                 status.statustype = status_type
 
-            # documents
-            internal_upload_enabled = ZaakTypeInformatieObjectTypeConfig.objects.get_visible_ztiot_configs_for_case(
-                self.case
-            ).exists()
-            external_upload_enabled = ZaakTypeConfig.objects.get_visible_zt_configs_for_case_type_identification(
-                self.case.zaaktype.identificatie
-            ).exists()
-
-            if external_upload_enabled:
-                external_upload_url = (
-                    ZaakTypeConfig.objects.get_visible_zt_configs_for_case_type_identification(
-                        self.case.zaaktype.identificatie
-                    )
-                    .get()
-                    .external_document_upload_url
-                )
-            else:
-                external_upload_url = ""
-
```

Comment:
I moved this to a function because this gnarly block clutters the get_context_data. It doesn't need anything from there, so it can get it's own space.